resources/assets/js/app.js

```diff
@@ -6,26 +6,11 @@
  */
 
 require('./bootstrap');
+require('jquery-tags-input/dist/jquery.tagsinput.min');
 
-/**
- * Next, we will create a fresh Vue application instance and attach it to
- * the page. Then, you may begin adding components to this application
- * or customize the JavaScript scaffolding to fit your unique needs.
- */
-
-//Vue.component('example', require('./components/people/dashboard/kids.vue'));
-
-const app = new Vue({
-    el: '#app',
-
-    data: {
-      activities_description_show: false,
-      reminders_frequency: 'once',
-      accept_invite_user: false,
-    },
-    methods: {
-    },
-});
+require('./tags');
+require('./search');
+require('./contacts');
 
```

Comment:
We need to add 
```
window.Vue = require('vue');

const app = new Vue({
    el: '#app',

    data: {
        activities_description_show: false,
        reminders_frequency: 'once',
        accept_invite_user: false,
    },
        methods: {
    },
});
```

here, otherwise we'll break some screens.

Reply:
Ok I wasn't aware that Vue was actually being used (since it's included in the default template) but I'll go ahead and make the changes

Reply:
I see they were added with the API

Reply:
Yeah it's especially used for Laravel Passport, and on some screens here and there.